plugins/colorbutton/plugin.js

```diff
@@ -315,8 +315,10 @@ CKEDITOR.plugins.add( 'colorbutton', {
 		/**
 		 * Selects the specified color in the specified panel block.
 		 *
-		 * @param block
-		 * @param color
+		 * @private
+		 * @member CKEDITOR.plugins.colorbutton
```

Comment:
This member doesn't exist. And `selectColor` is not exposed. In the past we used to [comment them with //](https://github.com/cksource/ckeditor-dev/blob/7454f54/core/style.js#L1195-L1197) - so that we have API docs in nice format, and at the same time they're not picked by JSDuck.

But it looks very bad. Instead let's just use `/*` (note no double asterix) for it. It should not be picked by JSDuck.